app/views/tag_wranglers/show.html.erb

```diff
@@ -17,6 +17,9 @@
       <p><%= ts("Your lack of fandoms makes the wrangulator sad. Fandoms can be assigned via the") %> <%= link_to ts('tag wranglers page'), tag_wranglers_path %>.</p>
     <% else %>
       <p><%= ts('Fandoms assigned to you via the ') %> <%= link_to ts('tag wranglers page'), tag_wranglers_path %>. <%= ts('The unwrangled count is the number of unwrangled tags which were used on works in the given fandom.') %></p>
+      <% if (@wrangler == @current_user || logged_in_as_admin?) && @wrangler.last_wrangling_activity %>
+        <p><%= ts("%{wrangler_login} last wrangled at ", wrangler_login: @wrangler.login) %> <span class="datetime"><%= time_in_zone(@wrangler.last_wrangling_activity.updated_at) %></span>.</p>
+      <% end %>
```

Comment:
I'm thinking yes, in case they were at one point but are no longer assigned any fandoms at the moment, but I'll defer to @sarken here

Reply:
Displaying it in that case sounds fine to me.

Reply:
Unfortunately, it looks like the issue actually specifies that the "NAME last wrangled at DATE" line should be below the "Fandoms assigned to you via the tag wranglers page" line, so it may be necessary to either have multiple copies of the line, or check `@fandoms.empty?` more than once. I don't like the repetition, but I don't know how else to do it.

(FYI, I'm not an official volunteer, so I don't have the power to mark things as resolved in your PR.)

Reply:
Argh, you're right. I think I could add a partial to save some duplication, but in this case I'm not sure that would be worth it

Reply:
I think the partial might be worth it (or changing the structure of the `if` statements), because it's going to be annoying to copy it over every time it needs to be changed.